tools/cprnc/CMakeLists.txt

```diff
@@ -1,24 +1,19 @@
 PROJECT(CPRNC C Fortran)
 
- 	
-execute_process(COMMAND ../configure --macros-format=CMake WORKING_DIRECTORY ${CMAKE_CURRENT_LIST_DIR})
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "$ENV{CIMEROOT}/src/CMake")
+# Generate this with: $cimeroot/tools/configure --mpilib=mpi-serial --macros-format=CMake
+#   You'll also need to source the .env_mach_specific.sh file before trying to build cprnc
+include("${PROJECT_BINARY_DIR}/Macros.cmake")
 
-INCLUDE(Macros.cmake)
+if (NOT NETCDF_PATH)
+  message(FATAL_ERROR "Requires NETCDF_PATH either from Macros.cmake or argument")
+endif()
 
 ENABLE_LANGUAGE(Fortran)
 
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8)
 
 SET(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} $ENV{CIMEROOT}/src/externals/pio2/cmake)
 
-# Defining NETCDF_LIBRARIES and NETCDF_INCLUDE_DIR enables bypassing
-#   the search for Netcdf
-IF (NOT DEFINED NETCDF_LIBRARIES OR NOT DEFINED NETCDF_INCLUDE_DIR)
-  SET(NETCDF_INCLUDE_DIR ${NETCDF_PATH}/include)
-  SET(NETCDF_LIBRARIES ${NETCDF_PATH}/lib/libnetcdff.a)
-ENDIF ()
-
 ADD_CUSTOM_COMMAND(
```

Comment:
This is a pre-existing issue, so I'm okay deferring it to a separate issue, though it feels more important now that cprnc may be built all the time as part of testing: I believe that the `ADD_CUSTOM_COMMAND` command should have a `DEPENDS` clause:

```
DEPENDS ${PROJECT_SOURCE_DIR}/compare_vars_mod.F90.in ${PROJECT_SOURCE_DIR}/../../src/externals/genf90/genf90.pl
```

Without that, `compare_vars_mod.F90` isn't regenerated even if its dependencies change.

Maybe connected with this: I don't understand the purpose of the `ADD_CUSTOM_TARGET` command on line 23. Does anyone understand why that's there and whether it's actually needed?

Again, I'm happy to open a separate issue for this if you don't want to muddy up this PR with this change.

Reply:
Ok I added the suggested `DEPENDS` clause and removed the mystery `ADD_CUSTOM_TARGET` command. Jim Foucar and I did not understand that `ADD_CUSTOM_TARGET` command too. I am running the regression tests with these changes.

Reply:
I don't see this change in the latest commit. Can you double-check that the changes you mention were actually committed?

Reply:
@WesCoomber

Reply:
@billsacks @jgfouca 
The change was on my local branch. I have pushed the commit with the appropriate changes to remote. 
c2f7b31af5418f6fad01a2aeb206b70028f5badb

Reply:
Thanks!